packages/fcl-wc/src/fcl-wc.test.js

```diff
@@ -1,8 +1,21 @@
-import * as fclWC from "./fcl-wc"
+import {config} from "@onflow/config"
 
 describe("Init Client", () => {
   it("should throw without projectId", async () => {
+    async function initWC() {
+      await config.overload(
+        {
+          "flow.network.default": "testnet",
+        },
+        fclWC.init
+      )
+    }
+
+    // Mock transport then import fcl-wc because startup of fcl will call getChainId util which hits the chain
+    config.put("sdk.transport", async ix => ix)
```

Comment:
Can we update to also use `config.overload` for this configuration variable as well?

Reply:
Done